src/Elastic.CommonSchema.NLog/EcsLayout.cs

```diff
@@ -86,6 +86,11 @@ public EcsLayout()
 		public Layout ServerAddress { get; set; }
 		public Layout ServerIp { get; set; }
 		public Layout ServerUser { get; set; }
+		/// <summary>
+        /// Optional action to enrich the constructed <see cref="Base">EcsEvent</see> before it is serialized
```

Comment:
Mind changes the spaces to tabs on this PR?

Reply:
Sure thing! Done with: https://github.com/ghayes7/ecs-dotnet/commit/da34b9dfa4fa1c55254066af5de61ddff9e7e50a